integration/simulation/network/geth_network.go

```diff
@@ -115,11 +115,15 @@ func (n *networkInMemGeth) Create(params *params.SimParams, stats *stats.Stats)
 func (n *networkInMemGeth) TearDown() {
 	for _, client := range n.obscuroClients {
 		temp := client
-		go (*temp).Call(nil, obscuroclient.RPCStopHost) //nolint:errcheck
-		go (*temp).Stop()
+		go func() {
+			_ = (*temp).Call(nil, obscuroclient.RPCStopHost)
+			(*temp).Stop()
+		}()
 	}
 
 	n.gethNetwork.StopNodes()
+	// double tap. todo: this is probably unnecessary
+	defer n.gethNetwork.StopNodes()
```

Comment:
@BedrockSquirrel I don't think it makes sense to defer  the final call in a method. This would make sense if you deferred `defer n.gethNetwork.StopNodes()` at the top of the method, in case stopping one of the clients blew up.